pkgs/servers/zookeeper/default.nix

```diff
@@ -47,8 +44,10 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     homepage = "https://zookeeper.apache.org";
     description = "Apache Zookeeper";
+    changelog = "https://zookeeper.apache.org/doc/r${version}/releasenotes.html";
     license = licenses.asl20;
     maintainers = with maintainers; [ nathan-gs pradeepchhetri ztzg ];
     platforms = platforms.unix;
+    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
```

Comment:
This should probably have `binaryBytecode` too.

Reply:
Adjusted, thanks! `binaryNativeCode` does not make sense here it's all `.jar`.

Reply:
I did actually see something that looks like a copy of libressl (?) when unzipping the jar, complete with multiple architectures, so assumed you'd seen that.